components/channel_view/channel_view.jsx

```diff
@@ -90,10 +90,12 @@ export default class ChannelView extends React.PureComponent {
             channelId: props.channelId,
             deferredPostView: ChannelView.createDeferredPostView(),
         };
+
+        this.channelView = React.createRef();
```

Comment:
Could you rename it to 
```suggestion
        this.channelViewRef = React.createRef();
```